libs/feature/record/src/lib/data-view-map/data-view-map.component.ts

```diff
@@ -111,9 +128,45 @@ export class DataViewMapComponent {
     private mdViewFacade: MdViewFacade,
     private mapUtils: MapUtilsService,
     private linkHelper: LinkHelperService,
-    private proxy: ProxyService
+    private proxy: ProxyService,
+    private featureInfo: FeatureInfoService,
+    private changeRef: ChangeDetectorRef,
+    private styleService: MapStyleService
   ) {}
 
+  ngOnDestroy(): void {
+    this.subscription.unsubscribe()
+  }
+
+  ngOnInit(): void {
+    this.selectionStyle = this.styleService.createDefaultStyle({
```

Comment:
why not something like `this.styleService.createHighlightStyle` instead? So that we don't need to read the app config again

Reply:
What do you mean ? a function that takes the default style and change something in it (like the color) ?
I usually do this but i'm not sure it is a use case for the `MapStyleService`.
Also, I see more _highlight_ term for the hover thing, than for the selection.

What would your function do ?

Reply:
just generate a style similar to the default one but with larger circles and the secondary color, for example